diff --git a/xbmc/utils/AMLUtils.cpp b/xbmc/utils/AMLUtils.cpp
index 995891b..5679867 100644
--- a/xbmc/utils/AMLUtils.cpp
+++ b/xbmc/utils/AMLUtils.cpp
@@ -507,7 +507,7 @@ bool aml_probe_resolutions(std::vector<RESOLUTION_INFO> &resolutions)
       if (aml_mode_to_resolution(i->c_str(), &res))
         resolutions.push_back(res);
 
-      if (aml_has_frac_rate_policy())
+      if (1)
       {
         // Add fractional frame rates: 23.976, 29.97 and 59.94 Hz
         switch ((int)res.fRefreshRate)
